templates/helm/values.yaml.tpl

```diff
@@ -28,6 +28,9 @@ deployment:
   # Which priorityClassName to set?
   # See: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority
   priorityClassName: ""
+
+role:
+ labels: {}
```

Comment:
Can we add a comment above the role section to explain that installing as `installScope: cluster` will modify its behaviour